generators/app/tasks/appSetup/androidProjectSetup.js

```diff
@@ -16,6 +16,18 @@ function addDotenvPluginAndRNScreen() {
     'if (enableHermes) {',
     "implementation 'androidx.appcompat:appcompat:1.2.0-alpha03'\n\timplementation 'androidx.multidex:multidex:2.0.1'\n\tif (enableHermes) {"
   );
+  updatedBuildGradleContent = updatedBuildGradleContent.replace(
+    'compileSdkVersion rootProject.ext.compileSdkVersion',
+    'compileSdkVersion rootProject.ext.compileSdkVersion\nflavorDimensions "buildtype"'
+  );
+  updatedBuildGradleContent = updatedBuildGradleContent.replace(
+    '// applicationVariants are e.g. debug, release',
+    'productFlavors {\nqa {\ndimension \'buildtype\'\napplicationIdSuffix ".qa"\n}\nstage {\ndimension \'buildtype\'\napplicationIdSuffix ".stage"\n}\nproduction {\ndimension \'buildtype\'\napplicationIdSuffix ".production"\n}\n}\n// applicationVariants are e.g. debug, release'
+  );
+  updatedBuildGradleContent = updatedBuildGradleContent.replace(
+    'versionName "1.0"',
+    `versionName "1.0"\nresValue "string", "build_config_package", "com.${this.projectName.toLowerCase()}"`
+  );
```

Comment:
Same. Also, you forgot to add the `\t` in all cases
<img width="628" alt="Captura de Pantalla 2020-04-02 a la(s) 12 58 47" src="https://user-images.githubusercontent.com/8583214/78275901-9f2d4400-74e8-11ea-8fd6-7d4e47113f6d.png">